### ISSUE TITLE
[Bug]: ResourcefulBees 1.16.4 Crashing whenever loading world

### ISSUE THREAD

**Issue by GaminExotic** [Resourceful-Bees/ResourcefulBees#305]
### Bug Description

When i installed ResourcefulBees in my modpack and tried to load my world it would crash when it would reach 100%

### Version

ResourcefulBees1.16.3-1.16.3-0.5.6b.jar

### Mod Loader Version

1.16.4

### Logs or additional context

---- Minecraft Crash Report ----
// Who set us up the TNT?

Time: 11/15/21 6:30 PM
Description: Watching Server

java.lang.Error: ServerHangWatchdog detected that a single server tick took 180.00 seconds (should be max 0.05)
	at java.util.AbstractList.removeRange(AbstractList.java:569) ~[?:1.8.0_51] {}
	at java.util.AbstractList.clear(AbstractList.java:234) ~[?:1.8.0_51] {}
	at net.minecraft.util.NonNullList.clear(SourceFile:76) ~[?:?] {re:mixin,re:classloading}
	at org.cursegame.minecraft.recycler.recipe.RecyclingRecipeManager.deleteDuplicates(RecyclingRecipeManager.java:176) ~[?:360] {re:classloading}
	at org.cursegame.minecraft.recycler.recipe.RecyclingManager.loadRecipes(RecyclingManager.java:120) ~[?:360] {re:classloading}
	at org.cursegame.minecraft.recycler.recipe.RecyclingManager.initialize(RecyclingManager.java:93) ~[?:360] {re:classloading}
	at org.cursegame.minecraft.recycler.ModRecycler.onServerStarted(ModRecycler.java:72) ~[?:360] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_842_ModRecycler_onServerStarted_FMLServerStartedEvent.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-4.0.0.jar:?] {}
	at net.minecraftforge.eventbus.EventBus$$Lambda$2621/960983277.invoke(Unknown Source) ~[?:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-4.0.0.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-4.0.0.jar:?] {}
	at net.minecraftforge.fml.server.ServerLifecycleHooks.handleServerStarted(ServerLifecycleHooks.java:107) ~[?:?] {re:classloading}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:621) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.lambda$startServer$0(MinecraftServer.java:232) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer$$Lambda$21499/1753059483.run(Unknown Source) ~[?:?] {}
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_51] {}

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Integrated Server Watchdog
Stacktrace:
	at java.util.AbstractList.removeRange(AbstractList.java:569) ~[?:1.8.0_51] {}
	at java.util.AbstractList.clear(AbstractList.java:234) ~[?:1.8.0_51] {}
	at net.minecraft.util.NonNullList.clear(SourceFile:76) ~[?:?] {re:mixin,re:classloading}
	at org.cursegame.minecraft.recycler.recipe.RecyclingRecipeManager.deleteDuplicates(RecyclingRecipeManager.java:176) ~[?:360] {re:classloading}
	at org.cursegame.minecraft.recycler.recipe.RecyclingManager.loadRecipes(RecyclingManager.java:120) ~[?:360] {re:classloading}
	at org.cursegame.minecraft.recycler.recipe.RecyclingManager.initialize(RecyclingManager.java:93) ~[?:360] {re:classloading}
	at org.cursegame.minecraft.recycler.ModRecycler.onServerStarted(ModRecycler.java:72) ~[?:360] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_842_ModRecycler_onServerStarted_FMLServerStartedEvent.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-4.0.0.jar:?] {}
	at net.minecraftforge.eventbus.EventBus$$Lambda$2621/960983277.invoke(Unknown Source) ~[?:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-4.0.0.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-4.0.0.jar:?] {}
	at net.minecraftforge.fml.server.ServerLifecycleHooks.handleServerStarted(ServerLifecycleHooks.java:107) ~[?:?] {re:classloading}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:621) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.lambda$startServer$0(MinecraftServer.java:232) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
-- Thread Dump --
Details:
	Threads: "Integrated Server Watchdog" Id=170 RUNNABLE
	at sun.management.ThreadImpl.dumpThreads0(Native Method)
	at sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:446)
	at com.performant.coremod.world.IntegratedServerHangWatchdog.run(IntegratedServerHangWatchdog.java:61)
	at java.lang.Thread.run(Thread.java:745)

"ChunkLoaderWorker-0" Id=156 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@76fb20f8
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@76fb20f8
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

"Thread-46" Id=140 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Server thread" Id=138 RUNNABLE
	at java.util.AbstractList.removeRange(AbstractList.java:569)
	at java.util.AbstractList.clear(AbstractList.java:234)
	at net.minecraft.util.NonNullList.clear(SourceFile:76)
	at org.cursegame.minecraft.recycler.recipe.RecyclingRecipeManager.deleteDuplicates(RecyclingRecipeManager.java:176)
	at org.cursegame.minecraft.recycler.recipe.RecyclingManager.loadRecipes(RecyclingManager.java:120)
	at org.cursegame.minecraft.recycler.recipe.RecyclingManager.initialize(RecyclingManager.java:93)
	at org.cursegame.minecraft.recycler.ModRecycler.onServerStarted(ModRecycler.java:72)
	at net.minecraftforge.eventbus.ASMEventHandler_842_ModRecycler_onServerStarted_FMLServerStartedEvent.invoke(.dynamic)
	...

"Snooper Timer" Id=139 WAITING on java.util.TaskQueue@783cc057
	at java.lang.Object.wait(Native Method)
	-  waiting on java.util.TaskQueue@783cc057
	at java.lang.Object.wait(Object.java:502)
	at java.util.TimerThread.mainLoop(Timer.java:526)
	at java.util.TimerThread.run(Timer.java:505)

"Sound engine" Id=137 WAITING on java.lang.String@7c034b88
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.lang.String@7c034b88
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at net.minecraft.client.audio.SoundEngineExecutor.func_223705_bi(SourceFile:49)
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213161_c(SourceFile:129)
	at net.minecraft.client.audio.SoundEngineExecutor.func_213178_c(SourceFile:42)
	at net.minecraft.client.audio.SoundEngineExecutor$$Lambda$14145/2108605572.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)

"Java2D Disposer" Id=113 WAITING on java.lang.ref.ReferenceQueue$Lock@3901ca6c
	at java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@3901ca6c
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:143)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:164)
	at sun.java2d.Disposer.run(Disposer.java:148)
	at java.lang.Thread.run(Thread.java:745)

"Thread-31" Id=88 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-30" Id=87 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-29" Id=75 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-28" Id=74 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-27" Id=73 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-26" Id=72 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-25" Id=71 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-24" Id=70 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-23" Id=69 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-22" Id=68 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-21" Id=61 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Narrator" Id=59 TIMED_WAITING
	at java.lang.Thread.sleep(Native Method)
	at com.mojang.text2speech.NarratorWindows$NarratorThread.run(NarratorWindows.java:84)
	at java.lang.Thread.run(Thread.java:745)

"Thread-17" Id=56 RUNNABLE (in native)
	at sun.nio.ch.Iocp.getQueuedCompletionStatus(Native Method)
	at sun.nio.ch.Iocp.access$300(Iocp.java:46)
	at sun.nio.ch.Iocp$EventHandlerTask.run(Iocp.java:333)
	at sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

	Number of locked synchronizers = 1
	- java.util.concurrent.ThreadPoolExecutor$Worker@2463c2cb

"Thread-16" Id=55 RUNNABLE (in native)
	at sun.nio.ch.Iocp.getQueuedCompletionStatus(Native Method)
	at sun.nio.ch.Iocp.access$300(Iocp.java:46)
	at sun.nio.ch.Iocp$EventHandlerTask.run(Iocp.java:333)
	at sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

	Number of locked synchronizers = 1
	- java.util.concurrent.ThreadPoolExecutor$Worker@12faa1c

"Thread-15" Id=54 RUNNABLE (in native)
	at sun.nio.ch.Iocp.getQueuedCompletionStatus(Native Method)
	at sun.nio.ch.Iocp.access$300(Iocp.java:46)
	at sun.nio.ch.Iocp$EventHandlerTask.run(Iocp.java:333)
	at sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

	Number of locked synchronizers = 1
	- java.util.concurrent.ThreadPoolExecutor$Worker@5a9e352b

"Thread-14" Id=53 RUNNABLE (in native)
	at sun.nio.ch.Iocp.getQueuedCompletionStatus(Native Method)
	at sun.nio.ch.Iocp.access$300(Iocp.java:46)
	at sun.nio.ch.Iocp$EventHandlerTask.run(Iocp.java:333)
	at sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

	Number of locked synchronizers = 1
	- java.util.concurrent.ThreadPoolExecutor$Worker@2bc986fd

"Thread-13" Id=52 RUNNABLE (in native)
	at sun.nio.ch.Iocp.getQueuedCompletionStatus(Native Method)
	at sun.nio.ch.Iocp.access$300(Iocp.java:46)
	at sun.nio.ch.Iocp$EventHandlerTask.run(Iocp.java:333)
	at sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

	Number of locked synchronizers = 1
	- java.util.concurrent.ThreadPoolExecutor$Worker@35221973

"Thread-12" Id=51 RUNNABLE (in native)
	at sun.nio.ch.Iocp.getQueuedCompletionStatus(Native Method)
	at sun.nio.ch.Iocp.access$300(Iocp.java:46)
	at sun.nio.ch.Iocp$EventHandlerTask.run(Iocp.java:333)
	at sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

	Number of locked synchronizers = 1
	- java.util.concurrent.ThreadPoolExecutor$Worker@e9aae1a

"Thread-11" Id=50 RUNNABLE (in native)
	at sun.nio.ch.Iocp.getQueuedCompletionStatus(Native Method)
	at sun.nio.ch.Iocp.access$300(Iocp.java:46)
	at sun.nio.ch.Iocp$EventHandlerTask.run(Iocp.java:333)
	at sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

	Number of locked synchronizers = 1
	- java.util.concurrent.ThreadPoolExecutor$Worker@4a67683f

"Thread-10" Id=49 RUNNABLE (in native)
	at sun.nio.ch.Iocp.getQueuedCompletionStatus(Native Method)
	at sun.nio.ch.Iocp.access$300(Iocp.java:46)
	at sun.nio.ch.Iocp$EventHandlerTask.run(Iocp.java:333)
	at sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

	Number of locked synchronizers = 1
	- java.util.concurrent.ThreadPoolExecutor$Worker@4134cd6

"Thread-9" Id=48 RUNNABLE (in native)
	at sun.nio.ch.Iocp.getQueuedCompletionStatus(Native Method)
	at sun.nio.ch.Iocp.access$300(Iocp.java:46)
	at sun.nio.ch.Iocp$EventHandlerTask.run(Iocp.java:333)
	at java.lang.Thread.run(Thread.java:745)

"Timer-0" Id=46 TIMED_WAITING on java.util.TaskQueue@16d2d154
	at java.lang.Object.wait(Native Method)
	-  waiting on java.util.TaskQueue@16d2d154
	at java.util.TimerThread.mainLoop(Timer.java:552)
	at java.util.TimerThread.run(Timer.java:505)

"Snooper Timer" Id=31 WAITING on java.util.TaskQueue@87295a1
	at java.lang.Object.wait(Native Method)
	-  waiting on java.util.TaskQueue@87295a1
	at java.lang.Object.wait(Object.java:502)
	at java.util.TimerThread.mainLoop(Timer.java:526)
	at java.util.TimerThread.run(Timer.java:505)

"Timer hack thread" Id=29 TIMED_WAITING
	at java.lang.Thread.sleep(Native Method)
	at net.minecraft.util.Util$6.run(SourceFile:636)

"Thread-2" Id=18 RUNNABLE (in native)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus0(Native Method)
	at sun.nio.fs.WindowsNativeDispatcher.GetQueuedCompletionStatus(WindowsNativeDispatcher.java:996)
	at sun.nio.fs.WindowsWatchService$Poller.run(WindowsWatchService.java:524)
	at java.lang.Thread.run(Thread.java:745)

"Thread-1" Id=17 TIMED_WAITING
	at sun.misc.Unsafe.park(Native Method)
	at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:338)
	at com.electronwill.nightconfig.core.file.FileWatcher$WatcherThread.run(FileWatcher.java:190)

"Attach Listener" Id=6 RUNNABLE

"Signal Dispatcher" Id=5 RUNNABLE

"Finalizer" Id=3 WAITING on java.lang.ref.ReferenceQueue$Lock@6e4bd02c
	at java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@6e4bd02c
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:143)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:164)
	at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:209)

"Reference Handler" Id=2 WAITING on java.lang.ref.Reference$Lock@592e3523
	at java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.Reference$Lock@592e3523
	at java.lang.Object.wait(Object.java:502)
	at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:157)

"Render thread" Id=1 RUNNABLE
	at org.lwjgl.opengl.GL11.glDisableClientState(Native Method)
	at com.mojang.blaze3d.platform.GlStateManager.func_227772_z_(GlStateManager.java:1255)
	at net.minecraft.client.renderer.vertex.VertexFormatElement$Usage.lambda$static$1(VertexFormatElement.java:136)
	at net.minecraft.client.renderer.vertex.VertexFormatElement$Usage$$Lambda$3850/1724962286.accept(Unknown Source)
	at net.minecraft.client.renderer.vertex.VertexFormatElement$Usage.func_227901_a_(VertexFormatElement.java:184)
	at net.minecraft.client.renderer.vertex.VertexFormatElement.func_227898_g_(VertexFormatElement.java:88)
	at net.minecraft.client.renderer.vertex.VertexFormat.func_227895_d_(VertexFormat.java:82)
	at net.minecraft.client.renderer.WorldVertexBufferUploader.func_227844_a_(SourceFile:47)
	...

Stacktrace:
	at com.performant.coremod.world.IntegratedServerHangWatchdog.run(IntegratedServerHangWatchdog.java:79) [?:1.16.2-5-3.72m] {re:classloading}
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_51] {}

-- Performance stats --
Details:
	Random tick rate: 100
	Level stats: ResourceKey[minecraft:dimension / minecraft:overworld]: players: 0, entities: 139 [quark:stoneling:64,quark:toretoise:26,minecraft:bat:15,minecraft:item:12,minecraft:iron_golem:9], block_entities: 2906 [mekanism:structural_glass:501,cyclic:item_pipe:309,botanypots:botany_pot:244,mekanismgenerators:turbine_casing:225,mekanism:thermal_evaporation_block:216], block_ticks: 61, fluid_ticks: 3, chunk_source: ServerChunkCache: 2025,
ResourceKey[minecraft:dimension / minecraft:the_nether]: players: 0, entities: 0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: ServerChunkCache: 0,
ResourceKey[minecraft:dimension / minecraft:the_end]: players: 0, entities: 0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: ServerChunkCache: 0,
ResourceKey[minecraft:dimension / compactmachines:compact_world]: players: 0, entities: 0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: ServerChunkCache: 0,
ResourceKey[minecraft:dimension / woot:tartarus]: players: 0, entities: 0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: ServerChunkCache: 0,
ResourceKey[minecraft:dimension / gaiadimension:gaia_dimension]: players: 0, entities: 0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: ServerChunkCache: 0,
ResourceKey[minecraft:dimension / allthemodium:mining]: players: 0, entities: 0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: ServerChunkCache: 0,
ResourceKey[minecraft:dimension / allthemodium:the_other]: players: 0, entities: 0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: ServerChunkCache: 0,
ResourceKey[minecraft:dimension / jamd:mining]: players: 0, entities: 0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: ServerChunkCache: 0

-- System Details --
Details:
	Minecraft Version: 1.16.4
	Minecraft Version ID: 1.16.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 4717235104 bytes (4498 MB) / 8522825728 bytes (8128 MB) up to 8522825728 bytes (8128 MB)
	CPUs: 8
	JVM Flags: 10 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmn128M -Xmx8127M -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	ModLauncher: 8.0.9+86+master.3cf110c
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.2.jar mixin PLUGINSERVICE 
		/eventbus-4.0.0.jar eventbus PLUGINSERVICE 
		/forge-1.16.4-35.1.37.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.16.4-35.1.37.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-3.0.1.jar accesstransformer PLUGINSERVICE 
		/forge-1.16.4-35.1.37.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.16.4-35.1.37.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.2.jar mixin TRANSFORMATIONSERVICE 
		/forge-1.16.4-35.1.37.jar fml TRANSFORMATIONSERVICE 
	FML: 35.1
	Forge: net.minecraftforge:35.1.37
	FML Language Providers: 
		javafml@35.1
		minecraft@1
		kotori_scala@2.13.4-build-2
	Mod List: 
		CyclopsCore-1.16.4-1.11.1.jar                     |Cyclops Core                  |cyclopscore                   |1.11.1              |DONE      |NOSIGNATURE
		AdditionalEnchantedMiner-1.16.5-16.1.9.jar        |QuarryPlus                    |quarryplus                    |16.1.9              |DONE      |1a:13:52:63:6f:dc:0c:ad:7f:8a:64:ac:46:58:8a:0c:90:ea:2c:5d:11:ac:4c:d4:62:85:c7:d1:00:fa:9c:76
		NetherPortalFix_1.16.3-7.2.1.jar                  |NetherPortalFix               |netherportalfix               |7.2.1               |DONE      |NOSIGNATURE
		refinedpipes-0.5.jar                              |Refined Pipes                 |refinedpipes                  |0.5                 |DONE      |NOSIGNATURE
		ExtendedCrafting-1.16.4-3.1.6.jar                 |Extended Crafting             |extendedcrafting              |3.1.6               |DONE      |NOSIGNATURE
		saveandloadinventories_1.16.4-1.1.jar             |Save and Load Inventories     |saveandloadinventories        |1.1                 |DONE      |NOSIGNATURE
		Industrial+Agriculture[Core]-1.16.3-1.7.1.jar     |Industrial Agriculture        |industrialagriculture         |1.16.3-1.7.1        |DONE      |NOSIGNATURE
		Neat 1.7-27.jar                                   |Neat                          |neat                          |1.7-27              |DONE      |NOSIGNATURE
		IronJetpacks-1.16.4-4.2.0.jar                     |Iron Jetpacks                 |ironjetpacks                  |4.2.0               |DONE      |NOSIGNATURE
		JRFTL [1.16.5]-1.2.jar                            |JRFTL                         |jrftl                         |1.2                 |DONE      |NOSIGNATURE
		CookingForBlockheads_1.16.3-9.2.2.jar             |Cooking for Blockheads        |cookingforblockheads          |9.2.2               |DONE      |NOSIGNATURE
		Placebo-1.16.4-4.5.0.jar                          |Placebo                       |placebo                       |4.5.0               |DONE      |NOSIGNATURE
		dankstorage-3.14.jar                              |Dank Storage                  |dankstorage                   |3.14                |DONE      |NOSIGNATURE
		citadel-1.5.3.jar                                 |Citadel                       |citadel                       |1.5.3               |DONE      |NOSIGNATURE
		NoFog-1.16.2-Forge-b1.jar                         |NoFog                         |nofog                         |1.0.1               |DONE      |NOSIGNATURE
		Powah-1.16.4-2.3.15.jar                           |Powah                         |powah                         |2.3.15              |DONE      |NOSIGNATURE
		rangedpumps-0.8.2.jar                             |Ranged Pumps                  |rangedpumps                   |0.8.2               |DONE      |NOSIGNATURE
		Wither-Skeleton-Tweaks-1.16.4-5.3.0.jar           |Wither Skeleton Tweaks        |wstweaks                      |5.3.0               |DONE      |NOSIGNATURE
		Mutant-Beasts-Mod-1.16.4.jar                      |Mutant Beasts                 |mutantbeasts                  |1.16.4-1.1.3        |DONE      |d9:be:bd:b6:9a:e4:14:aa:05:67:fb:84:06:77:a0:c5:10:ec:27:15:1b:d6:c0:88:49:9a:ef:26:77:61:0b:5e
		Bookshelf-1.16.4-9.3.25.jar                       |Bookshelf                     |bookshelf                     |9.3.25              |DONE      |eb:c4:b1:67:8b:f9:0c:db:dc:4f:01:b1:8e:61:64:39:4c:10:85:0b:a6:c4:c7:48:f0:fa:95:f2:cb:08:3a:e5
		randompatches-2.4.4-forge.jar                     |RandomPatches                 |randompatches                 |2.4.4-forge         |DONE      |92:f6:29:d4:09:89:f5:f5:98:5e:20:34:31:d0:7b:58:22:06:bd:a5:d1:6a:92:6e:ac:3d:8d:18:c5:b2:5b:d7
		ChanceCubes-1.16.3-5.0.2.394.jar                  |Chance Cubes                  |chancecubes                   |1.16.3-5.0.2.394    |DONE      |NOSIGNATURE
		buildinggadgets-1.16.4-3.7.3.jar                  |Building Gadgets              |buildinggadgets               |3.7.3               |DONE      |NOSIGNATURE
		DarkUtilities-1.16.4-7.0.8.jar                    |Dark Utilities                |darkutils                     |7.0.8               |DONE      |eb:c4:b1:67:8b:f9:0c:db:dc:4f:01:b1:8e:61:64:39:4c:10:85:0b:a6:c4:c7:48:f0:fa:95:f2:cb:08:3a:e5
		Apotheosis-1.16.4-4.5.3.jar                       |Apotheosis                    |apotheosis                    |4.5.3               |DONE      |NOSIGNATURE
		MekanismGenerators-1.16.4-10.0.19.446.jar         |Mekanism: Generators          |mekanismgenerators            |10.0.19             |DONE      |NOSIGNATURE
		Hwyla-forge-1.10.11-B78_1.16.2.jar                |Waila                         |waila                         |1.10.11-B78_1.16.2  |DONE      |NOSIGNATURE
		FpsReducer-mc1.16.5-1.19.jar                      |FPS Reducer                   |fpsreducer                    |mc1.16.5-1.19       |DONE      |NOSIGNATURE
		carryon-1.16.5-1.15.3.13.jar                      |Carry On                      |carryon                       |1.15.3.13           |DONE      |8c:03:ac:7d:21:62:65:e2:83:91:f3:22:57:99:ed:75:78:1e:db:de:03:99:ef:53:3b:59:95:18:01:bc:84:a9
		JustEnoughResources-1.16.4-0.12.0.113.jar         |Just Enough Resources         |jeresources                   |0.12.0.113          |DONE      |NOSIGNATURE
		shetiphiancore-1.16-3.8.6.jar                     |ShetiPhian-Core               |shetiphiancore                |3.8.6               |DONE      |NOSIGNATURE
		mob_grinding_utils-1.16.5-0.4.23.jar              |Mob Grinding Utils            |mob_grinding_utils            |1.16.5-0.4.23       |DONE      |NOSIGNATURE
		refinedstorage-1.9.12.jar                         |Refined Storage               |refinedstorage                |1.9.12              |DONE      |NOSIGNATURE
		PrettyPipes-1.9.5.jar                             |Pretty Pipes                  |prettypipes                   |1.9.5               |DONE      |NOSIGNATURE
		JAMD-1.1.4-beta+mc-1.16.4.jar                     |Just Another Mining Dimension |jamd                          |1.1.4-beta+mc-1.16.4|DONE      |23:8f:95:0f:65:ec:2e:38:99:79:f4:bc:47:8a:0b:df:29:ef:2d:82:66:20:09:20:02:dc:4a:15:97:45:f8:43
		industrial-foregoing-1.16.4-3.2.8.2-6b5c95b.jar   |Industrial Foregoing          |industrialforegoing           |3.2.8.2             |DONE      |NOSIGNATURE
		torchmaster-2.3.5-alpha.jar                       |Torchmaster                   |torchmaster                   |2.3.5-alpha         |DONE      |NOSIGNATURE
		ResourcefulBees1.16.3-1.16.3-0.5.6b.jar           |Resourceful Bees              |resourcefulbees               |1.16.3-0.5.6b       |DONE      |NOSIGNATURE
		MekanismAdditions-1.16.4-10.0.19.446.jar          |Mekanism: Additions           |mekanismadditions             |10.0.19             |DONE      |NOSIGNATURE
		valkyrielib-1.16.4-3.0.6.1.jar                    |ValkyrieLib                   |valkyrielib                   |3.0.6.1             |DONE      |NOSIGNATURE
		envirocore-1.16.4-3.0.6.0.jar                     |Environmental Core            |envirocore                    |3.0.6.0             |DONE      |NOSIGNATURE
		envirotech-1.16.4-3.0.6.1.jar                     |Environmental Tech            |envirotech                    |3.0.6.0             |DONE      |NOSIGNATURE
		Lollipop-1.16.4-3.2.8.jar                         |Lollipop                      |lollipop                      |3.2.8               |DONE      |NOSIGNATURE
		Iron-Furnaces-Mod-1.16.4.jar                      |Iron Furnaces                 |ironfurnaces                  |2.5.5               |DONE      |NOSIGNATURE
		ChunkAnimator-1.16.5-1.2.4.jar                    |Chunk Animator                |chunkanimator                 |1.16.5-1.2.4        |DONE      |NOSIGNATURE
		supermartijn642corelib-1.0.12-mc1.16.5.jar        |SuperMartijn642's Core Lib    |supermartijn642corelib        |1.0.12              |DONE      |NOSIGNATURE
		Botania-1.16.4-414.jar                            |Botania                       |botania                       |1.16.4-414          |DONE      |NOSIGNATURE
		MysticalAdaptations-1.16.4-1.1.1.jar              |Mystical Adaptations          |mysticaladaptations           |1.16.4-1.1.1        |DONE      |NOSIGNATURE
		gaiadimension-1.16.4-1.3.1.jar                    |The Gaia Dimension            |gaiadimension                 |1.3.1               |DONE      |NOSIGNATURE
		Patchouli-1.16.4-53.2.jar                         |Patchouli                     |patchouli                     |1.16.4-53.2         |DONE      |NOSIGNATURE
		collective-1.16.4-1.53.jar                        |Collective                    |collective                    |1.53                |DONE      |NOSIGNATURE
		OreExcavation-1.8.157.jar                         |Ore Excavation                |oreexcavation                 |1.8.157             |DONE      |e7:68:1c:0d:b9:7e:cf:f8:f3:40:9c:84:c5:39:d7:a4:59:78:b0:6b:c3:fd:b7:4f:69:18:a3:88:e3:76:8c:3f
		time-in-a-bottle-1.1.0.jar                        |Time In A Bottle              |tiab                          |1.1.0               |DONE      |NOSIGNATURE
		elevatorid-1.16.5-1.7.13.jar                      |Elevator Mod                  |elevatorid                    |1.16.5-1.7.13       |DONE      |NOSIGNATURE
		Runelic-1.16.4-6.0.5.jar                          |Runelic                       |runelic                       |6.0.5               |DONE      |eb:c4:b1:67:8b:f9:0c:db:dc:4f:01:b1:8e:61:64:39:4c:10:85:0b:a6:c4:c7:48:f0:fa:95:f2:cb:08:3a:e5
		MekanismTools-1.16.4-10.0.19.446.jar              |Mekanism: Tools               |mekanismtools                 |10.0.19             |DONE      |NOSIGNATURE
		architectury-1.24.35-forge.jar                    |Architectury                  |architectury                  |1.24.35             |DONE      |NOSIGNATURE
		ftb-gui-library-1605.2.1.41-forge.jar             |FTB GUI Library               |ftbguilibrary                 |1605.2.1.41         |DONE      |NOSIGNATURE
		Chargers-1.16.5-3.1.0.5.jar                       |Chargers                      |chargers                      |3.1.0.5             |DONE      |15:a4:a3:0e:d2:f8:64:0c:a4:97:30:5f:48:ba:5f:69:f9:23:42:14:a8:5a:60:3a:b1:b1:29:1a:0a:37:3c:79
		curios-forge-1.16.5-4.0.5.3.jar                   |Curios API                    |curios                        |1.16.5-4.0.5.3      |DONE      |NOSIGNATURE
		angelring-1.16.5-1.3.4.1.jar                      |Angel Ring                    |angelring                     |1.3.4.1             |DONE      |NOSIGNATURE
		compact-crafting-1.0.0-alpha.2.jar                |Compact Crafting              |compactcrafting               |1.0.0-alpha.2       |DONE      |NOSIGNATURE
		cloth-config-4.11.26-forge.jar                    |Cloth Config v4 API           |cloth-config                  |4.11.26             |DONE      |NOSIGNATURE
		trashcans-1.0.10-mc1.16.5.jar                     |Trash Cans                    |trashcans                     |1.0.10              |DONE      |NOSIGNATURE
		charm-forge-1.16.4-2.1.1-hotfix3.jar              |Charm                         |charm                         |2.1.1               |DONE      |NOSIGNATURE
		CodeChickenLib-1.16.4-3.5.1.408-universal.jar     |CodeChicken Lib               |codechickenlib                |3.5.1.408           |DONE      |31:e6:db:63:47:4a:6e:e0:0a:2c:11:d1:76:db:4e:82:ff:56:2d:29:93:d2:e5:02:bd:d3:bd:9d:27:47:a5:71
		GameMenuModOption-1.16.4-1.8.jar                  |Game Menu Mod Option          |gamemenumodoption             |1.8                 |DONE      |NOSIGNATURE
		QuantumStorage-1.16.4-5.3.0.jar                   |QuantumStorage                |quantumstorage                |5.3.0Alpha5         |DONE      |NOSIGNATURE
		Cyclic-1.16.5-1.5.8.jar                           |Cyclic                        |cyclic                        |1.16.5-1.5.8        |DONE      |1f:47:ac:b1:61:82:96:b8:47:19:16:d2:61:81:11:60:3a:06:4b:61:31:56:7d:44:31:1e:0c:6f:22:5b:4c:ed
		QuarkOddities-1.16.3.jar                          |Quark Oddities                |quarkoddities                 |1.16.3              |DONE      |NOSIGNATURE
		mowziesmobs-1.5.19.jar                            |Mowzie's Mobs                 |mowziesmobs                   |1.5.19              |DONE      |NOSIGNATURE
		SimplyJetpacks2-1.16.4-2.6.6.0.jar                |Simply Jetpacks 2             |simplyjetpacks                |1.16.4-2.6.6.0      |DONE      |NOSIGNATURE
		Cucumber-1.16.4-4.1.10.jar                        |Cucumber Library              |cucumber                      |4.1.10              |DONE      |NOSIGNATURE
		woot-1.16.5-1.0.8.0.jar                           |Woot                          |woot                          |1.16.5-1.0.8.0      |DONE      |NOSIGNATURE
		vanillaexcavators-1.16.2-2.1.3.jar                |Vanilla Excavators            |vanillaexcavators             |2.1.3               |DONE      |NOSIGNATURE
		Bountiful+Baubles+FORGE-1.16.3-0.0.2.jar          |Bountiful Baubles             |bountifulbaubles              |NONE                |DONE      |NOSIGNATURE
		jei-1.16.4-7.6.1.74.jar                           |Just Enough Items             |jei                           |7.6.1.74            |DONE      |NOSIGNATURE
		jei-professions-1.0.0-1.16.4.jar                  |JEI Professions               |jeiprofessions                |1.0.0               |DONE      |NOSIGNATURE
		Mekanism-1.16.4-10.0.19.446.jar                   |Mekanism                      |mekanism                      |10.0.19             |DONE      |NOSIGNATURE
		reap-1.16.4-1.0.3.jar                             |Reap Mod                      |reap                          |1.16.4-1.0.3        |DONE      |NOSIGNATURE
		WAWLA-1.16.4-7.0.4.jar                            |WAWLA                         |wawla                         |7.0.4               |DONE      |eb:c4:b1:67:8b:f9:0c:db:dc:4f:01:b1:8e:61:64:39:4c:10:85:0b:a6:c4:c7:48:f0:fa:95:f2:cb:08:3a:e5
		create-mc1.16.5_v0.3.1c.jar                       |Create                        |create                        |mc1.16.5_v0.3.1c+151|DONE      |NOSIGNATURE
		Waystones_1.16.5-7.4.0.jar                        |Waystones                     |waystones                     |7.4.0               |DONE      |NOSIGNATURE
		Clumps-6.0.0.15.jar                               |Clumps                        |clumps                        |6.0.0.15            |DONE      |NOSIGNATURE
		journeymap-1.16.5-5.7.3.jar                       |Journeymap                    |journeymap                    |5.7.3               |DONE      |NOSIGNATURE
		NaturesCompass-1.16.4-1.8.6.jar                   |Nature's Compass              |naturescompass                |1.16.4-1.8.6        |DONE      |NOSIGNATURE
		compactmachines-4.0.0-beta.2.jar                  |Compact Machines 4            |compactmachines               |4.0.0-beta.2        |DONE      |NOSIGNATURE
		BotanyPots-1.16.4-6.0.17.jar                      |BotanyPots                    |botanypots                    |6.0.17              |DONE      |eb:c4:b1:67:8b:f9:0c:db:dc:4f:01:b1:8e:61:64:39:4c:10:85:0b:a6:c4:c7:48:f0:fa:95:f2:cb:08:3a:e5
		VanillaTweaks-1.16.5-1.5.40.jar                   |VanillaTweaks                 |vanillatweaks                 |1.16.5-1.5.40       |DONE      |NOSIGNATURE
		curioofundying-forge-1.16.4-5.1.0.0.jar           |Curio of Undying              |curioofundying                |1.16.4-5.1.0.0      |DONE      |NOSIGNATURE
		FarmingForBlockheads_1.16.3-7.3.0.jar             |Farming for Blockheads        |farmingforblockheads          |7.3.0               |DONE      |NOSIGNATURE
		mcjtylib-1.16-5.0.16.jar                          |McJtyLib                      |mcjtylib                      |1.16-5.0.16         |DONE      |NOSIGNATURE
		rftoolsbase-1.16-2.0.10.jar                       |RFToolsBase                   |rftoolsbase                   |1.16-2.0.10         |DONE      |NOSIGNATURE
		xnet-1.16-3.0.11.jar                              |XNet                          |xnet                          |1.16-3.0.11         |DONE      |NOSIGNATURE
		notenoughwands-1.16-3.0.1.jar                     |Not Enough Wands              |notenoughwands                |1.16-3.0.1          |DONE      |NOSIGNATURE
		ClickMachine-1.16.4-4.4.0.jar                     |Click Machine                 |clickmachine                  |4.4.0               |DONE      |NOSIGNATURE
		iceandfire-2.1.5-1.16.4.jar                       |Ice and Fire                  |iceandfire                    |2.1.5-1.16.4        |DONE      |NOSIGNATURE
		Toast-Control-1.16.4-4.3.1.jar                    |Toast Control                 |toastcontrol                  |4.3.1               |DONE      |NOSIGNATURE
		inventorypets-1.16.5-2.0.14.jar                   |Inventory Pets                |inventorypets                 |2.0.14              |DONE      |NOSIGNATURE
		mininggadgets-1.7.1.jar                           |Mining Gadgets                |mininggadgets                 |1.7.1               |DONE      |NOSIGNATURE
		EnderStorage-1.16.4-2.7.1.166-universal.jar       |EnderStorage                  |enderstorage                  |2.7.1.166           |DONE      |31:e6:db:63:47:4a:6e:e0:0a:2c:11:d1:76:db:4e:82:ff:56:2d:29:93:d2:e5:02:bd:d3:bd:9d:27:47:a5:71
		extradisks-1.16.4-1.5.1.jar                       |Extra Disks                   |extradisks                    |1.5.1               |DONE      |NOSIGNATURE
		forge-1.16.4-35.1.37-universal.jar                |Forge                         |forge                         |35.1.37             |DONE      |22:af:21:d8:19:82:7f:93:94:fe:2b:ac:b7:e4:41:57:68:39:87:b1:a7:5c:c6:44:f9:25:74:21:14:f5:0d:90
		WailaHarvestability-mc1.16.x-forge-1.1.15.jar     |Waila Harvestability          |wailaharvestability           |1.1.15              |DONE      |NOSIGNATURE
		ColossalChests-1.16.4-1.7.13.jar                  |ColossalChests                |colossalchests                |1.7.13              |DONE      |NOSIGNATURE
		MysticalAgriculture-1.16.4-4.2.0.jar              |Mystical Agriculture          |mysticalagriculture           |4.2.0               |DONE      |NOSIGNATURE
		MysticalAgradditions-1.16.4-4.1.5.jar             |Mystical Agradditions         |mysticalagradditions          |4.1.5               |DONE      |NOSIGNATURE
		ironchest-1.16.4-11.2.10.jar                      |Iron Chests                   |ironchest                     |1.16.4-11.2.10      |DONE      |NOSIGNATURE
		ZeroCore2-1.16.4-2.0.21.jar                       |Zero CORE 2                   |zerocore                      |1.16.4-2.0.21       |DONE      |NOSIGNATURE
		ExtremeReactors2-1.16.4-2.0.23.jar                |Extreme Reactors 2            |bigreactors                   |1.16.4-2.0.23       |DONE      |NOSIGNATURE
		forge-1.16.4-35.1.37-client.jar                   |Minecraft                     |minecraft                     |1.16.4              |DONE      |NOSIGNATURE
		cofh_core-1.16.4-1.2.1.jar                        |CoFH Core                     |cofh_core                     |1.2.1               |DONE      |NOSIGNATURE
		thermal_foundation-1.16.4-1.2.0.jar               |Thermal Series                |thermal                       |1.2.0               |DONE      |NOSIGNATURE
		thermal_expansion-1.16.4-1.2.0.jar                |Thermal Expansion             |thermal_expansion             |1.2.0               |DONE      |NOSIGNATURE
		Lucky-Block-Mod-1.16.4.jar                        |Lucky Block                   |lucky                         |1.16.4-3            |DONE      |NOSIGNATURE
		rftoolsutility-1.16-3.0.22.jar                    |RFToolsUtility                |rftoolsutility                |1.16-3.0.22         |DONE      |NOSIGNATURE
		itlt-1.16.x-2.0.1.jar                             |It's The Little Things        |itlt                          |2.0.1               |DONE      |NOSIGNATURE
		swingthroughgrass-1.16.4-1.5.3.jar                |SwingThroughGrass             |swingthroughgrass             |1.16.4-1.5.3        |DONE      |NOSIGNATURE
		MouseTweaks-2.14-mc1.16.2.jar                     |Mouse Tweaks                  |mousetweaks                   |2.14                |DONE      |NOSIGNATURE
		titanium-1.16.5-3.2.8.4-10.jar                    |Titanium                      |titanium                      |3.2.8.4             |DONE      |NOSIGNATURE
		avaritialite-1.4.2-1.16.4.jar                     |Avaritia Lite                 |avaritia                      |1.4.1               |DONE      |NOSIGNATURE
		ImmersiveEngineering-1.16.4-4.1.2-129.jar         |Immersive Engineering         |immersiveengineering          |1.16.4-4.1.2-129    |DONE      |44:39:94:cf:1d:8c:be:3c:7f:a9:ee:f4:1e:63:a5:ac:61:f9:c2:87:d5:5b:d9:d6:8c:b5:3e:96:5d:8e:3f:b7
		morevanillalib-1.16.2-1.2.14.jar                  |MoreVanillaLib                |morevanillalib                |1.2.14              |DONE      |NOSIGNATURE
		allthemodium-1.2.3-1.16.4-35.1.13.jar             |Allthemodium                  |allthemodium                  |1.2.3-1.16.4-35.1.13|DONE      |NOSIGNATURE
		ChickenChunks-1.16.4-2.7.0.85-universal.jar       |ChickenChunks                 |chickenchunks                 |2.7.0.85            |DONE      |31:e6:db:63:47:4a:6e:e0:0a:2c:11:d1:76:db:4e:82:ff:56:2d:29:93:d2:e5:02:bd:d3:bd:9d:27:47:a5:71
		LargeFluidTank-16.1.8.jar                         |FluidTank                     |fluidtank                     |16.1.8              |DONE      |1a:13:52:63:6f:dc:0c:ad:7f:8a:64:ac:46:58:8a:0c:90:ea:2c:5d:11:ac:4c:d4:62:85:c7:d1:00:fa:9c:76
		Corail-Recycler-Mod-1.16.4.jar                    |Corail Recycler NE            |corail_recycler_ne            |1.16.4-360          |DONE      |NOSIGNATURE
		jeiintegration_1.16.4-6.1.1.11.jar                |JEI Integration               |jeiintegration                |6.1.1.11            |DONE      |NOSIGNATURE
		rats-7.2.0-1.16.5.jar                             |Rats                          |rats                          |7.2.0               |DONE      |NOSIGNATURE
		AutoRegLib-1.6-47.jar                             |AutoRegLib                    |autoreglib                    |1.6-47              |DONE      |NOSIGNATURE
		Quark-r2.4-305.jar                                |Quark                         |quark                         |r2.4-305            |DONE      |NOSIGNATURE
		StorageDrawers-1.16.3-8.3.0.jar                   |Storage Drawers               |storagedrawers                |8.3.0               |DONE      |NOSIGNATURE
		FluxNetworks-1.16.5-6.1.7.12.jar                  |Flux Networks                 |fluxnetworks                  |6.1.7.12            |DONE      |NOSIGNATURE
		performant-1.16.2-5-3.72m.jar                     |Performant                    |performant                    |3.56m               |DONE      |NOSIGNATURE
		cagedmobs-1.16.4-1.2.6.jar                        |Caged Mobs                    |cagedmobs                     |1.16.4-1.2.6        |DONE      |NOSIGNATURE
		UltimateSolarPanel-1.16.4-1.3.1.jar               |Ultimate Solar Panels         |ultimatesolarpanels           |1.3.0               |DONE      |NOSIGNATURE
		OreGrowRF-1.16.4-1.0.0.3.jar                      |Ore Grow EX                   |ore_grow_ex                   |1.0.3               |DONE      |NOSIGNATURE
		Chlorine-1.16.5-0.0.1.3.jar                       |Chlorine                      |chlorine                      |0.0.1.3             |DONE      |NOSIGNATURE
		SpawnEggRecipes - 1.8.0(Huge recipe fix)(EASY MODE|Spawn Egg Recipes Mod         |spawn                         |1.16.4-1.2.0        |DONE      |NOSIGNATURE
		refinedstorageaddons-0.7.2.jar                    |Refined Storage Addons        |refinedstorageaddons          |0.7.2               |DONE      |NOSIGNATURE
		overloadedarmorbar-5.1.0.jar                      |Overloaded Armor Bar          |overloadedarmorbar            |5.1.0               |DONE      |NOSIGNATURE
		balancedenchanting-1.0.jar                        |Balanced Enchanting           |balancedenchanting            |1.0                 |DONE      |NOSIGNATURE
	Crash Report UUID: 078c7f3d-0cdb-4f04-bdc0-ec3e4b309eaf
	Patchouli open book context: n/a
	Player Count: 0 / 8; []
	Data Packs: resourcefulbees:internals, vanilla, mod:forge, mod:apotheosis (incompatible), mod:placebo (incompatible), mod:jei, mod:jeiintegration, mod:vanillatweaks, mod:mousetweaks, mod:bookshelf, mod:darkutils (incompatible), mod:neat (incompatible), mod:clumps, mod:jrftl (incompatible), mod:balancedenchanting (incompatible), mod:cyclopscore, mod:quarkoddities (incompatible), mod:cucumber, mod:extendedcrafting, mod:reap, mod:autoreglib (incompatible), mod:wstweaks (incompatible), mod:quark (incompatible), mod:shetiphiancore, mod:codechickenlib (incompatible), mod:enderstorage (incompatible), mod:colossalchests, mod:cyclic (incompatible), mod:ironfurnaces, mod:corail_recycler_ne, mod:cloth-config (incompatible), mod:mekanism, mod:mutantbeasts (incompatible), mod:mekanismgenerators, mod:refinedstorage, mod:cofh_core (incompatible), mod:thermal (incompatible), mod:industrialforegoing (incompatible), mod:titanium (incompatible), mod:mekanismadditions, mod:valkyrielib, mod:envirocore (incompatible), mod:envirotech, mod:thermal_expansion (incompatible), mod:mekanismtools, mod:chargers, mod:refinedstorageaddons, mod:quarryplus, mod:netherportalfix (incompatible), mod:woot, mod:vanillaexcavators (incompatible), mod:bountifulbaubles (incompatible), mod:refinedpipes, mod:ironjetpacks, mod:create, mod:citadel (incompatible), mod:naturescompass (incompatible), mod:powah, mod:rangedpumps, mod:botanypots, mod:chancecubes (incompatible), mod:mcjtylib, mod:rftoolsbase, mod:notenoughwands (incompatible), mod:iceandfire (incompatible), mod:waila, mod:inventorypets (incompatible), mod:prettypipes (incompatible), mod:mysticalagriculture, mod:mysticalagradditions, mod:zerocore, mod:jamd (incompatible), mod:lucky (incompatible), mod:morevanillalib (incompatible), mod:lollipop, mod:chickenchunks (incompatible), mod:gaiadimension (incompatible), mod:storagedrawers (incompatible), mod:fluxnetworks, mod:cagedmobs, mod:ultimatesolarpanels, mod:curios, mod:bigreactors, mod:quantumstorage (incompatible), mod:mob_grinding_utils, mod:torchmaster (incompatible), mod:avaritia, mod:ore_grow_ex, mod:spawn, mod:rats, mod:mowziesmobs (incompatible), mod:clickmachine (incompatible), mod:tiab, mod:nofog (incompatible), mod:chunkanimator, mod:performant (incompatible), mod:chlorine, mod:industrialagriculture (incompatible), mod:cookingforblockheads (incompatible), mod:ftbguilibrary (incompatible), mod:mysticaladaptations (incompatible), mod:farmingforblockheads (incompatible), mod:saveandloadinventories, mod:dankstorage, mod:randompatches, mod:buildinggadgets, mod:fpsreducer, mod:carryon, mod:jeresources, mod:supermartijn642corelib, mod:botania, mod:patchouli (incompatible), mod:collective, mod:oreexcavation, mod:elevatorid, mod:runelic, mod:architectury, mod:angelring (incompatible), mod:compactcrafting (incompatible), mod:trashcans, mod:charm (incompatible), mod:gamemenumodoption, mod:simplyjetpacks, mod:jeiprofessions (incompatible), mod:wawla, mod:waystones (incompatible), mod:journeymap (incompatible), mod:compactmachines, mod:curioofundying, mod:xnet, mod:toastcontrol (incompatible), mod:mininggadgets (incompatible), mod:extradisks, mod:wailaharvestability, mod:ironchest (incompatible), mod:rftoolsutility, mod:itlt, mod:swingthroughgrass (incompatible), mod:immersiveengineering (incompatible), mod:allthemodium, mod:fluidtank, mod:overloadedarmorbar (incompatible), mod:resourcefulbees (incompatible)
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'forge'

### Acknowledgement

- [X] I agree that I read the terms at the top of the page.

**Comment by Epic428** [comment]
That seems to be an issue with Minecraft. There’s not a single thing in that report that identifies us as the root cause

**Comment by GaminExotic** [comment]
> That seems to be an issue with Minecraft. There’s not a single thing in that report that identifies us as the root cause

how do i fix it?

**Comment by Epic428** [comment]
Well you could try loading the world again and see if you get the same error. You could try removing our mod and loading the world. And you could try creating a new world. But nothing in that report ever mentions a single thing from our mod as being part of the issue.

**Comment by ThatGravyBoat** [comment]
Closing as it doesn't seem to have anything to do with our mod. If it does just comment and ill reopen

**Comment by GaminExotic** [comment]
when resourcefulbees isnt installed it works perfectly but when i put
resourceful bees it crashes when loading a world sry for the late replu

**Comment by Epic428** [comment]
![5A9411BE-635D-4290-AC1F-F2078251FA1E](https://user-images.githubusercontent.com/291578/144875934-d2384ad4-9f62-4675-9d0f-ed6f804d0a30.jpeg)
Use 1.16.5 and the latest mod version. I’m not going to provide support for an outdated version.

**Comment by GaminExotic** [comment]
i am using 1.16.4 so i dont think i should use 1.16.5 for 1.16.4 but i will
try and let you know

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by Epic428** [comment]
1.16.5 literally added two methods to stop server crashes. There’s a bug that can be exploited which will crash loop your server. So every mod you have should work on 1.16.5. There’s literally NO REASON 1.16.4 should be used. Also I’m not providing support for a mod version that’s about 20+ versions behind

**Comment by GaminExotic** [comment]
i made my modpack like when 1.16.4 was release and it works and whenever i
try to make a new modpack it just crashes so yeah
also i tried 1.16.5 and it says that minecraft 1.16.5 is required

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by Epic428** [comment]
Well of course Minecraft 1.16.5 is needed. You need to update everything. Again. I’m not providing support for a mod version that’s 6 months+ out of date.

**Comment by GaminExotic** [comment]
ik but thanks for kinda helping me

**Comment by Epic428** [comment]
Idk if you’re being sarcastic or not. But the reason I won’t provide support for an outdated version is there’s highly likely a bug has been fixed in one of our newer versions or in a newer version of one of your other mods that fixes the issue. There’s no point in wasting time debugging an issue only to find out it’s already been fixed as a side effect of something else. If you update everything to latest and still have an issue then open up a new issue on github and we’ll look into it.
